catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorTest.java

```diff
@@ -114,7 +120,7 @@ public void testUpdateCallback() {
   }
 
   @Test
-  public void testRouteCreationWithoutContentComponent() throws Exception {
+  public void testRouteCreationWithoutContentComponent() {
```

Comment:
✏️ It shouldn't matter to leave a `throws Exception` on test methods.

Reply:
Is there a compelling reason to leave `throws Exception` on the test method even though no code throws one?

Reply:
just makes it simpler when maintaining since in any case, an exception being thrown out of a test is typically meant to fail the test unless the test is annotated to say otherwise. So leaving it simply avoids the perpetual remove/add/remove/add of the throws clause. I wouldn't do that in production code but for test ...